debezium-connector-postgres/RDS.md

```diff
@@ -66,7 +66,7 @@ This is where all the _magic_ happens where we put tie together all the pieces o
 Remember this version should match the version of database parameters you cloned in the prior steps.
 3. Select the **Template** of interest, I typically picked **Free tier** because it's free ;)
 4. Provide your database with a **DB Instance Identifier**.
-5. Under **Credentials Settings**, manually assign a **Master Password** and confirm it.
+5. Under **Credentials Settings**, manually assign a **RDS Master Password** and confirm it.
```

Comment:
AWS calls it `RDS Master Password` so I handled it like a name. we need to monitor if AWS updates the name to avoid divisive language too and then we can update to the new name

Reply:
👍